src/pages/shop/Shop.tsx

```diff
@@ -1,15 +1,193 @@
+                      </li>
+                      <li>
+                        <Link to='/'></Link>
+                      </li>
+                    </ul>
+                    </div>
+                    
+                  </div>
+                  <div className="productTags">
+                    <p>Product Tags</p>
+                    <Stack direction='column'>
+                           <Wrap spacing={10}>
+                              <WrapItem>
+                                <Button className='btn'><span>Bosch</span></Button>
+                              </WrapItem>
+                              <WrapItem>
+                                <Button className='btn'><span>Corkcicle</span></Button>
+                              </WrapItem>
+                              <WrapItem>
+                                <Button className='btn'><span>Handmade</span></Button>
+                              </WrapItem>
+                              <WrapItem>
+                                <Button className='btn'><span>Kinto</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Lether</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Lifestyle</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Orbitkey</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Pet</span></Button>
+                              </WrapItem>  <WrapItem>
+                                <Button className='btn'><span>Soma</span></Button>
+                              </WrapItem>
+                             </Wrap>
+                      </Stack>
+                  </div>
+                  <div className="productColor">
+                    <p>Product Color</p>
+                    <div className="color-groups">
+                      <button className="color colorOne"></button>
+                      <button className="color colorTwo"></button>
+                      <button className="color colorThere"></button>
+                      <button className="color colorFour"></button>
+                      <button className="color colorFive"></button>
+                      <button className="color colorSix"></button>
+                      <button className="color colorSeven"></button>
+                      <button className="color colorEight"></button>
+                      <button className="color colorNine"></button>
+                      <button className="color colorTen"></button>
+                      <button className="color colorEleven"></button>
+                      <button className="color colorTwelve"></button>
+                      <button className="color colorThereteen lastest"></button>
```

Comment:
just use the same approach here: declare an array with colors and just do .map

and color-black, color-nude, etc. much more better than colorOne and colorTwo =)